monasca-api-python/Dockerfile

```diff
@@ -34,6 +34,7 @@ ENV CONFIG_TEMPLATE=true \
   KEYSTONE_ADMIN_PASSWORD=secretadmin \
   KEYSTONE_ADMIN_TENANT=admin \
   KEYSTONE_ADMIN_DOMAIN=default \
+  INSECURE=false \
```

Comment:
Change variable name to `KEYSTONE_INSECURE`

Reply:
ok,thanks.